connector/src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -146,6 +148,23 @@ object SparkBigQueryOptions {
           .stripMargin.replace('\n', ' '))
     }
     val readDataFormat = DataFormat.valueOf(readDataFormatParam)
+
+    val arrowEnableUnsafeMemoryAccessProperty = "arrow.enable_unsafe_memory_access"
+    val arrowEnableUnsafeMemoryAccess = getAnyBooleanOption(normalizedAllConf,
+      parameters,
+      arrowEnableUnsafeMemoryAccessProperty,
+      DefaultArrowUnsafeMemoryAccess)
+    System.setProperty(arrowEnableUnsafeMemoryAccessProperty,
```

Comment:
This will be set on the driver process only, not on the executors. The property needs to be set in the ArrowBinaryIterator/Reader as well (and same for the other property)

Reply:
Added it in BigQueryRDD as well